src/Graphics/Image.hs

```diff
@@ -288,13 +290,23 @@ normalize img =
     !iMin = minVal img
 {-# INLINE normalize #-}
 
--- -- | Check weather two images are equal within a tolerance. Useful for comparing
--- -- images with `Float` or `Double` precision.
--- eqTol
---   :: (ColorModel cs e, Ord e) =>
---      e -> Image cs e -> Image cs e -> Bool
--- eqTol !tol !img1 = IP.and . thresholdWith2 (eqTolPx tol) img1
--- {-# INLINE eqTol #-}
+-- | Check weather two images are equal within a tolerance. Useful for comparing
+-- images with `Float` or `Double` precision.
+--
+-- >>> eqTol 0.99 (makeImage (Sz2 2 2) (const 0) :: Image Model.Y Float) (makeImage (Sz2 2 2) (const 1))
+-- False
+--
+-- >>> eqTol 1 (makeImage (Sz2 2 2) (const 0) :: Image Model.Y Float) (makeImage (Sz2 2 2) (const 1))
+-- True
+eqTol
+  :: (ColorModel cs e, Ord e) =>
+     e -> Image cs e -> Image cs e -> Bool
+eqTol !tol !img1 =
+  IP.and . thresholdWith2 thresholdPixel img1
+  where
+    thresholdPixel pixelA pixelB = F.foldl' (||) False (thresholdComponent <$> pixelA <*> pixelB)
```

Comment:
```suggestion
    thresholdPixel pixelA pixelB = F.foldl' (&&) True (thresholdComponent <$> pixelA <*> pixelB)
```
Every channel in the pixel should be equal up to the tolerance, so I think it should be a conjunction, right?

Reply:
:+1:

Reply:
I fixed it :) and good catch